src/main/java/com/esri/samples/displayinformation/dictionary_renderer_graphics_overlay/DictionaryRendererGraphicsOverlaySample.java

```diff
@@ -39,6 +39,11 @@
 import javafx.scene.layout.StackPane;
 import javafx.stage.Stage;
 
+/**
+ * Demonstrates how to apply a dictionary renderer to a graphics overlay and display mil2525d graphics. The dictionary
+ * renderer creates these graphics using a local mil2525d style file and a XML file with key-value attributes for each
+ * graphic.
+ */
```

Comment:
We don't put descriptions on classes right now. Please check if the readme needs to be updated for these changes.